pkgpanda/__init__.py

```diff
@@ -792,10 +797,10 @@ def _get_service_names(_dir):
             if package.username is not None:
                 sysusers.add_user(package.username, package.group)
 
-            # Ensure the state directory in `/var/lib/dcos` exists
+            # Ensure the state directory exists
             # TODO(cmaloney): On upgrade take a snapshot?
             if self.__manage_state_dir:
-                state_dir_path = '/var/lib/dcos/{}'.format(package.name)
+                state_dir_path = self.__state_dir_root + '/' + package.name
```

Comment:
+1, but I'll leave this since this module is already inconsistent about how it joins filesystem paths, as well as the entire repo. I filed a ticket to track cleaning that up: https://jira.mesosphere.com/browse/DCOS_OSS-1270